tools/interop_matrix/create_matrix_images.py

```diff
@@ -39,7 +39,7 @@
 _LANGUAGES = client_matrix.LANG_RUNTIME_MATRIX.keys()
 # All gRPC release tags, flattened, deduped and sorted.
 _RELEASES = sorted(list(set(
-    i for l in client_matrix.LANG_RELEASE_MATRIX.values() for i in l)))
+    i['version'] for l in client_matrix.LANG_RELEASE_MATRIX.values() for i in l)))
```

Comment:
nit: apparently single character identifiers aren't considered pythonic enough according to the codestyle.

Reply:
A little confused, I wasn't aware of a style guide or formatter for the python under `tools` in our repo, is there one?